src/main/build/debug.h

```diff
@@ -87,6 +87,7 @@ typedef enum {
     DEBUG_DYN_IDLE,
     DEBUG_FEEDFORWARD_LIMIT,
     DEBUG_FEEDFORWARD,
+    DEBUG_ANGLE_CONTROLLER,
```

Comment:
Best put after `DEBUG_GPS_DOP` at the end of the list.
Also add `ANGLE_CONTROLLER` to the end of the list in debug.c.